src/components/about/AboutComponent.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react'
```

Comment:
missing semi colon (in every import)

Reply:
missing semi colon is part of the linter rules from google